extensions/positron-r/src/provider.ts

```diff
@@ -181,6 +135,57 @@ export async function* rRuntimeDiscoverer(): AsyncGenerator<positron.LanguageRun
 	}
 }
 
+async function getBinaries(): Promise<DiscoveredBinaries> {
+	// If the override paths are specified, use them exclusively
+	const overrideBinaries = discoverOverrideBinaries();
+	if (!!overrideBinaries) {
+		return { binaries: overrideBinaries, currentBinary: undefined };
+	}
+
+	// Consult various sources of R binaries
```

Comment:
Here's where I feel like some helpful comments have actually been removed, when this got extracted into a helper (I'm supportie of the helper).

Reply:
Update: I now see that some of this got transferred to the docstring above the helper. So now I'm just asking that you conform that any wisdom in those comments got preserved and landed in what seems like the most useful place for code maintenance.